proto/context/src/main/proto/spine/validation/field_views.proto

```diff
@@ -11,17 +11,30 @@ option java_multiple_files = true;
 
 import "spine/protodata/ast.proto";
 
+// Uniquely identifies a field declaration within the compilation messages.
```

Comment:
I think this type addresses not only "compilation" messages. It's a part of AST, so I say we shouldn't be specific about "compilation" messages.